src/tacacs/bash_tacplus/bash_tacplus.c

```diff
@@ -224,20 +227,40 @@ int tacacs_authorization(
     return result;
 }
 
+/*
+ * Get current SSH session remote address from environment variable
+ */
+void get_remote_address(char* dst, size_t size)
+{
+    // SSHD will create environment variable SSH_CONNECTION after user session created.
+    char *ssh_connection = getenv("SSH_CONNECTION");
+    if (ssh_connection != NULL) {
+        snprintf(dst, size, "%s", ssh_connection);
```

Comment:
>snprintf [](http://example.com/codeflow?start=8&length=8)

snprintf return value could be used to detect not enough dst size. Could you take use of it?

Reply:
Fixed